environment.yml

```diff
@@ -29,10 +29,11 @@ dependencies:
 - traits>=4.6.0
 - pyface>=6
 - traitsui>=6
+- testpath<0.4
 - pip:
   - mne
   - mayavi
-  - "https://api.github.com/repos/nipy/PySurfer/zipball/master"
+  - PySurfer[save_movie]
   - nitime
```

Comment:
these fixes were stolen from @larsoner :-)-

Reply:
Hmm ... but why do we even have Pysurfer in our environment.yml?

Reply:
I thought you added this ;-)

Reply:
I think it's an overkill to have an environment for mne-bids.

mne can work with only numpy, scipy and matplotlib. You should only require this for mne-bids and point to the mne environment for the full possible features.

Reply:
+1, let's discuss this here: https://github.com/mne-tools/mne-bids/issues/109

and move on with this PR